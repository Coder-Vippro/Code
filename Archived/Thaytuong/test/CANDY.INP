6
100
4
9
5
6
98