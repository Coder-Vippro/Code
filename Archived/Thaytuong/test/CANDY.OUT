3 114
1
2
3
3 108
4
5
6
